fvm/src/call_manager/default.rs

```diff
@@ -125,7 +125,16 @@ where
         self.call_stack_depth -= 1;
 
         if self.machine.context().tracing {
-            self.exec_trace.push(ExecutionEvent::Return(result.clone()));
+            self.exec_trace.push(ExecutionEvent::Return(match result {
+                Err(ref e) => Err(match e {
```

Comment:
This combination of `ref`s and `clone`s appeased the checker, but I feel like there's something better that can be done?

Reply:
Not really...

Reply:
Well, you can use `&result` and remove the `ref e` and friends, but you still need to clone. On the other hand, this will only happen when tracing is enabled.